crypto/evp/evp_enc.c

```diff
@@ -1837,7 +2024,7 @@ static void *evp_cipher_from_algorithm(const int name_id,
             break;
         }
     }
-    if ((fnciphcnt != 0 && fnciphcnt != 3 && fnciphcnt != 4)
+    if ((fnciphcnt != 0 && fnciphcnt != 3 && fnciphcnt != 4 && fnciphcnt != 6)
```

Comment:
@openssl/committers what should be the logic here? Which callback combinations are mandatory to implement?

Reply:
Fixed in my latest commit, resolving